p4-16/spec/P4-16-spec.mdk

```diff
@@ -3489,6 +3489,11 @@ Bit-strings also support the following operations:
   The effect of this statement is to set bits `H` through `L` (inclusive of
   both) of `e` to the bit-pattern represented by `x`, and leaves all other bits
   of `e` unchanged.  A slice of an unsigned integer is an unsigned integer.
+  A slice may also be specified as `[L+:W]` where L is the lowest bit of the slice
+  and W is the width of the slice.  In this case, only W must be a non-negative
+  local compile-time known numeric value.  L must also be a numeric value and
```

Comment:
Please add markup to this occurrence of L, i.e. `L` with backticks

Reply:
Also for the earlier occurrence of W in the paragraph.

Reply:
The `int<W>` description was just a copy of the whole paragraph from here, so I shortened that to just refer to the bitstring description.